pandas/tests/io/json/test_pandas.py

```diff
@@ -1032,6 +1038,102 @@ def test_to_jsonl(self):
         assert result == expected
         assert_frame_equal(pd.read_json(result, lines=True), df)
 
+    def test_readjson_chunks(self, lines_json_df):
+        """Basic test that read_json(chunks=True) gives the same result as
+        read_json(chunks=False)"""
+        # GH17048: memory usage when lines=True
+
+        for cs in [1, 1.0]:
```

Comment:
a float is not a valid chunksize or do we allow this if it's equal to an int?
we shouldn't do that

Reply:
We use _validate_integer to be consistent with the API for [chunksize in read_csv](https://github.com/pandas-dev/pandas/blob/master/pandas/io/parsers.py#L407).